### DISCUSSION TITLE
Move existing jellyseerr docker over to dockage

### DISCUSSION THREAD

**Discussion by LordSith77** [louislam/dockge#570] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

HP Proliant DL380 G7
Ubuntu Linux 20.04.6
I'm trying to move my existing jellyseerr docker image over to dockage. At first, it seemed to be working (I moved the whole folder from the original location to opt/stacks). Today it was throwing jellyfin API errors. I'm assuming I need to update the yml file for the ./config but not exactly sure where to point it (I'm not real fluent with dockers... just learning about them as I go). Currently, my yml is:
`version: '3'

services:

  jellyseerr:

    image: fallenbagel/jellyseerr:latest

    container_name: jellyseerr

    environment:

      - LOG_LEVEL=debug

      - TZ=America/Chicago

    ports:

      - 5055:5055

    volumes:

      - ./config:/app/config

    restart: unless-stopped`

The original location of my docker was ~/dockers/jellyseerr/ & ~/dockers/jellyseerr/config

Anyone know where I should point the .config: to (I'm trying to move this without losing all the data, as there's a ton of requests already in place that I don't want to lose).
Thanks for any and all help.

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Ubuntu Linux 20.04.6

### 🌐 Browser

Google Chrome 126.0.6478.127

### 🐋 Docker Version

24.0.7

**Comment by trythatagain** [comment]
I would try pointing  './config' to the full path of the config folder
